### PR TITLE
vscode-extensions.svsool.markdown-memo: 0.3.8 -> 0.3.9

### PR DESCRIPTION
**Pull request by ratsclub**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/vscode-extensions/default.nix [hunk 1]
```diff
@@ -1418,7 +1418,7 @@ let
 
       llvm-org.lldb-vscode = llvmPackages_8.lldb;
 
-      WakaTime.vscode-wakatime = callPackage ./wakatime { };
+      WakaTime.vscode-wakatime = callPackage ./wakatime {};
```
Comment (SuperSandro2000) [inline pkgs/misc/vscode-extensions/default.nix:1421]:
```suggestion
      WakaTime.vscode-wakatime = callPackage ./wakatime { };
```

File: pkgs/misc/vscode-extensions/default.nix [hunk 2]
```diff
@@ -1317,7 +1317,7 @@ let
         };
       };
 
-      ms-vsliveshare.vsliveshare = callPackage ./ms-vsliveshare-vsliveshare { };
+      ms-vsliveshare.vsliveshare = callPackage ./ms-vsliveshare-vsliveshare {};
```
Comment (SuperSandro2000) [inline pkgs/misc/vscode-extensions/default.nix:1320]:
```suggestion
      ms-vsliveshare.vsliveshare = callPackage ./ms-vsliveshare-vsliveshare { };
```

File: pkgs/misc/vscode-extensions/default.nix [hunk 3]
```diff
@@ -1303,7 +1303,7 @@ let
         };
       };
 
-      vadimcn.vscode-lldb = callPackage ./vscode-lldb { };
+      vadimcn.vscode-lldb = callPackage ./vscode-lldb {};
```
Comment (SuperSandro2000) [inline pkgs/misc/vscode-extensions/default.nix:1306]:
```suggestion
      vadimcn.vscode-lldb = callPackage ./vscode-lldb { };
```

File: pkgs/misc/vscode-extensions/default.nix [hunk 4]
```diff
@@ -1013,7 +1013,7 @@ let
         };
       };
 
-      matklad.rust-analyzer = callPackage ./rust-analyzer { };
+      matklad.rust-analyzer = callPackage ./rust-analyzer {};
```
Comment (SuperSandro2000) [inline pkgs/misc/vscode-extensions/default.nix:1016]:
```suggestion
      matklad.rust-analyzer = callPackage ./rust-analyzer { };
```

File: pkgs/misc/vscode-extensions/default.nix [hunk 5]
```diff
@@ -919,12 +919,12 @@ let
...
 
-      ms-vscode.cpptools = callPackage ./cpptools { };
+      ms-vscode.cpptools = callPackage ./cpptools {};
 
-      ms-vscode-remote.remote-ssh = callPackage ./remote-ssh { };
+      ms-vscode-remote.remote-ssh = callPackage ./remote-ssh {};
 
       ms-python.python = callPackage ./python {
-        extractNuGet = callPackage ./python/extract-nuget.nix { };
+        extractNuGet = callPackage ./python/extract-nuget.nix {};
```
Comment (SuperSandro2000) [inline pkgs/misc/vscode-extensions/default.nix:927]:
```suggestion
        extractNuGet = callPackage ./python/extract-nuget.nix { };
```

File: pkgs/misc/vscode-extensions/default.nix [hunk 6]
```diff
@@ -919,12 +919,12 @@ let
         };
       };
 
-      ms-vscode.cpptools = callPackage ./cpptools { };
+      ms-vscode.cpptools = callPackage ./cpptools {};
 
-      ms-vscode-remote.remote-ssh = callPackage ./remote-ssh { };
+      ms-vscode-remote.remote-ssh = callPackage ./remote-ssh {};
```
Comment (SuperSandro2000) [inline pkgs/misc/vscode-extensions/default.nix:924]:
```suggestion
      ms-vscode-remote.remote-ssh = callPackage ./remote-ssh { };
```

File: pkgs/misc/vscode-extensions/default.nix [hunk 7]
```diff
@@ -919,12 +919,12 @@ let
         };
       };
 
-      ms-vscode.cpptools = callPackage ./cpptools { };
+      ms-vscode.cpptools = callPackage ./cpptools {};
```
Comment (SuperSandro2000) [inline pkgs/misc/vscode-extensions/default.nix:922]:
```suggestion
      ms-vscode.cpptools = callPackage ./cpptools { };
```

File: pkgs/misc/vscode-extensions/default.nix [hunk 8]
```diff
@@ -905,7 +905,7 @@ let
         };
       };
 
-      ms-dotnettools.csharp = callPackage ./ms-dotnettools-csharp { };
+      ms-dotnettools.csharp = callPackage ./ms-dotnettools-csharp {};
```
Comment (SuperSandro2000) [inline pkgs/misc/vscode-extensions/default.nix:908]:
```suggestion
      ms-dotnettools.csharp = callPackage ./ms-dotnettools-csharp { };
```

File: pkgs/misc/vscode-extensions/default.nix [hunk 9]
```diff
@@ -671,7 +671,7 @@ let
         };
       };
 
-      hashicorp.terraform = callPackage ./terraform { };
+      hashicorp.terraform = callPackage ./terraform {};
```
Comment (SuperSandro2000) [inline pkgs/misc/vscode-extensions/default.nix:674]:
```suggestion
      hashicorp.terraform = callPackage ./terraform { };
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Oh well. I thought I can just commit the changes. Please revert the formatting changes.

**Comment by ratsclub** [comment]
Should I rename my commit?

**Pull request closed without merging**